app/build.gradle

```diff
@@ -116,8 +116,7 @@ dependencies {
     implementation 'com.mikhaellopez:circularprogressbar:3.0.3'
     implementation 'org.jsoup:jsoup:1.14.1'
     //    https://gitlab.com/eyeo/distpartners/libadblockplus-android/-/tags
-    implementation 'org.adblockplus:adblock-android-webview:5.0.1'
```

Comment:
What's the need to downgrade?

Reply:
Given version was not resolving properly (I tried to investigate it but with no result). That version builds fine

Reply:
This caused issues with AdBlock feature or functioning of the app overall?

Reply:
Gradle couldn't find this version and download it

Reply:
Android Studio suggessted me to update to 7.3.1 along with other libraries

Reply:
I already had AGP v. 7.2.2 and it definitely builds with that version. But as I mentioned before - it does not find `adblock-android-webview 5.0.1` (even though it is present in the repo) but it builds fine with the 4.4.0.
After upgrading Gradle to 7.4 and AGP to 7.3.1 it still fails due to `Failed to resolve: org.adblockplus:adblock-android-webview:5.0.1`

Reply:
Does it build successfully for you with `5.0.1`?

Reply:
Understandable. The app uses many old libraries and functionalities that easily breaks upon upgrading. I got to know about these by the logs in CI/CD.

Reply:
maybe it's due to repo havingt only 4.4.0 listed
https://mvnrepository.com/artifact/org.adblockplus/adblock-android-webview?repo=jcenter

Reply:
I just tried to build and yes, yes it failed now. I wonder why and how did that failed as I was able to build it yesterday as well as day before yesterday. 
> maybe it's due to repo havingt only 4.4.0 listed

This IMO might be one of the possible reason

Reply:
Me too. But I just wanted to hot fix stuff in order to build the repo. There was no reasoning behind decisions I made apart from successful build. You can change it however you like, though.